### PR TITLE
PMM-6038 PT summary.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

File: management/action/ptsummary_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+	actionOK, err := client.Default.Actions.GetAction(&actions.GetActionParams{
+		Context: pmmapitests.Context,
+		Body: actions.GetActionBody{
+			ActionID: explainActionOK.Payload.ActionID,
+		},
+	})
+	require.NoError(t, err)
+	require.Empty(t, actionOK.Payload.Error)
+	t.Log(actionOK.Payload.Output)
```
Comment (percona-csalguero) [inline management/action/ptsummary_test.go:34]: Shouldn't we check for not-empty ?

Comment (JiriCtvrtka) [reply management/action/ptsummary_test.go:35]: Yes, we should. Added.

File: management/action/ptsummary_test.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+	"github.com/percona/pmm/api/managementpb/json/client"
+	"github.com/percona/pmm/api/managementpb/json/client/actions"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestPTSummary(t *testing.T) {
+	t.Skip("not implemented yet")
```
Comment (BupycHuk) [inline management/action/ptsummary_test.go:15]: btw we can test it on `pmm-server`

File: management/action/ptsummary_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	"testing"
+	"time"
+
+	"github.com/percona/pmm/api/managementpb/json/client"
+	"github.com/percona/pmm/api/managementpb/json/client/actions"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPTSummary(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```
Comment (BupycHuk) [inline management/action/ptsummary_test.go:14]:
```suggestion
	ctx, cancel := context.WithTimeout(pmmapitests.Context, 30*time.Second)
```

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Is FB ready?

File: management/action/ptsummary_test.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	"testing"
+	"time"
+
+	"github.com/percona/pmm/api/managementpb/json/client"
+	"github.com/percona/pmm/api/managementpb/json/client/actions"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPTSummary(t *testing.T) {
+	ctx, cancel := context.WithTimeout(pmmapitests.Context, 30*time.Second)
```
Comment (BupycHuk) [inline management/action/ptsummary_test.go:14]: it's not imported

**Review by BupycHuk — APPROVED** [review]

**Comment by JiriCtvrtka** [comment]
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1065

**Pull request merged**